### PR TITLE
Added ords to the output for SORTED and SORTED_SET doc values

### PR DESCRIPTION
**Pull request by noofherder**

### CODE REVIEW

**Comment by noofherder** [comment]
TODO - add an endpoint to return ords and values without docids (and with a value filter?)

File: src/main/java/com/github/flaxsearch/resources/DocValuesResource.java [hunk 1]
```diff
@@ -130,6 +136,68 @@ else if (dvtype == DocValuesType.SORTED_SET) {
         return response;
     }
 
+    @Path("/ordered")
+    @GET
```
Comment (romseygeek) [inline src/main/java/com/github/flaxsearch/resources/DocValuesResource.java:140]: Can we make this ```/ordered/{field}``` to keep it in line with the other endpoints?  Plus it enforces that you need a field name for it to work.

File: src/test/java/com/github/flaxsearch/resources/TestDocValuesResource.java [hunk 2]
```diff
@@ -74,4 +74,54 @@ public void testAnyDocValuesEncoding() {
     	assertThat(values.get("1")).isEqualTo("c29tZSBieXRlcw==");
     }
 
+    @Test
+    public void testSortedSetValues() {
+        AnyDocValuesResponse response = resource.client().target("/docvalues/field4").request()
+                .get(new GenericType<AnyDocValuesResponse>() {});
```
Comment (romseygeek) [inline src/test/java/com/github/flaxsearch/resources/TestDocValuesResource.java]: You can use ```.get(AnyDocValuesResponse.class)``` here - ```GenericType``` is only needed for generics (eg Map or List).

**Review by romseygeek — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/flaxsearch/resources/DocValuesResource.java (hunk 1 above)
Comment (romseygeek) [reply src/main/java/com/github/flaxsearch/resources/DocValuesResource.java:140]: Oh, never mind, it's a subpath.  Ignore me!

Comment (noofherder) [reply src/main/java/com/github/flaxsearch/resources/DocValuesResource.java:140]: Ignored with great prejudice.

File: src/test/java/com/github/flaxsearch/resources/TestDocValuesResource.java (hunk 2 above)
Comment (noofherder) [reply src/test/java/com/github/flaxsearch/resources/TestDocValuesResource.java]: Ta! Fixed.

**Review by romseygeek — APPROVED** [review]

**Pull request merged**